packages/edit-site/src/components/sidebar-navigation-screen-library/index.js

```diff
@@ -57,13 +60,21 @@ export default function SidebarNavigationScreenLibrary() {
 			footer={
 				<ItemGroup>
```

Comment:
A small issue I found not related to this PR but could be good to fix here:
![image](https://github.com/WordPress/gutenberg/assets/7753001/b6ccb679-65d4-48fd-9671-40d911182798)

I'm seeing the footer separator even though there's nothing in the footer. Maybe we can move the `isMobileViewport` condition a level higher to prevent this?

Reply:
Thanks, I've tweaked this in https://github.com/WordPress/gutenberg/pull/51961/commits/dd3ae521b54ccef80471616325360e51a3df3ee8